TC/iocBoot/iocTC-IOC-01/st-common.lua

```diff
@@ -47,7 +47,7 @@ function twincat_stcommon_main()
 		iocsh.devMotorCreateAxis(motor_port, axis_num-1, plc_version)
 		iocsh.dbLoadRecords(single_axis_db, db_args)
 
-		status_args = string.format("P=%s,M=%s,IOCNAME=%s", pv_prefix, motor_pv, ioc_name)
+		status_args = string.format("P=%s,M=MOT:%s,IOCNAME=%s", pv_prefix, motor_pv, ioc_name)
```

Comment:
This seems like a bigger change - was this meant to be committed along with this branch? Or was it just for testing etc?

Reply:
I noticed motorStatus was not being loaded in by TC, but should be